home-dev/launch/docker-compose-apitests.yml

```diff
@@ -0,0 +1,136 @@
+version: "3.8"
```

Comment:
separate docker compose YML is created for running API tests. it repeats the original, but has 2 changes:
- CQRS is disabled for API tests (otherwise they would fail due to not implemented query params and reader-writer sync time)
- job period is decreased to make tests run faster (from 5 sec to 1 sec)